services/web/server/src/simcore_service_webserver/login/utils.py

```diff
@@ -158,27 +159,35 @@ async def send_mail(app: web.Application, msg: MIMEText):
     smtp_args = dict(
         hostname=cfg.SMTP_HOST,
         port=cfg.SMTP_PORT,
-        use_tls=cfg.SMTP_TLS_ENABLED,
+        use_tls=cfg.SMTP_PROTOCOL == EmailProtocol.TLS,
+        start_tls=cfg.SMTP_PROTOCOL == EmailProtocol.STARTTLS,
     )
     log.debug("Sending email with smtp configuration: %s", pformat(smtp_args))
     if cfg.SMTP_PORT == 587:
         # NOTE: aiosmtplib does not handle port 587 correctly
         # plaintext first, then use starttls
         # this is a workaround
         smtp = aiosmtplib.SMTP(**smtp_args)
-        await smtp.connect(use_tls=False, port=cfg.SMTP_PORT)
-        if cfg.SMTP_TLS_ENABLED:
-            log.info("Starting TLS ...")
+        if cfg.SMTP_PROTOCOL == EmailProtocol.STARTTLS:
+            log.debug("Unencrypted connection attempt to mailserver ...")
+            await smtp.connect(use_tls=False, port=cfg.SMTP_PORT)
+            log.debug("Starting STARTTLS ...")
+            log.warning(
+                "Certificates are not validated for STARTTLS (as it happens for TLS) on port %s!",
+                str(cfg.SMTP_PORT),
```

Comment:
tip: 
```python
f"{cfg.SMTP_PORT=}"
```
this will show as cfg.SMTP_PORT=thevalue

Also in the message you might want to add some kind of suggestion on how to fix this? This is going to print every time a mail is sent I believe.

Reply:
yep, you should use this

Reply:
done